src/app/fdctl/config/default.toml

```diff
@@ -11,11 +11,11 @@ name = "fd1"
 # or as root, so that it can configure kernel bypass networking.  Once
 # this configuration has been performed, the process will enter a highly
 # restrictive sandbox, drop all privileges, and switch to the user given
-# here.  When running the configuration steps of `fdctl configure`, data
-# will be permissioned as read/write for this user, irrespective of
-# which user is performing the configuration.
+# here.  When running the configuration steps of `fdctl configure` data
+# will be permissioned so that is it writable for this user and not the
+# user which is performing the configuration.
 #
-# Firedancer requires nothing from this user, and it should be as
+# Firedancer requires nothing from this user and it should be as
```

Comment:
```suggestion
# Firedancer requires nothing from this user, and it should be as
```
I think the comma is right here because "it should be as minimally permissioned as possible" is a whole sentence.